pkg/controller/node/nodecontroller.go

```diff
@@ -388,28 +455,48 @@ func (nc *NodeController) Run() {
 			defer nc.evictorLock.Unlock()
 			for k := range nc.zonePodEvictor {
 				nc.zonePodEvictor[k].Try(func(value TimedValue) (bool, time.Duration) {
-					obj, exists, err := nc.nodeStore.GetByKey(value.Value)
-					if err != nil {
-						glog.Warningf("Failed to get Node %v from the nodeStore: %v", value.Value, err)
-					} else if !exists {
-						glog.Warningf("Node %v no longer present in nodeStore!", value.Value)
+					// TODO(kevin-wangzefeng): flag-gate this part to disable alpha feature forgiveness by default.
+					if true {
+						obj, exists, err := nc.nodeStore.GetByKey(value.Value)
+						if err != nil {
+							glog.Warningf("Failed to get Node %v from the nodeStore: %v", value.Value, err)
+						} else if !exists {
+							glog.Warningf("Node %v no longer present in nodeStore!", value.Value)
+						} else {
+							node, _ := obj.(*api.Node)
+							zone := utilnode.GetZoneKey(node)
+							EvictionsNumber.WithLabelValues(zone).Inc()
```

Comment:
Now that I think about it, this should be incremented in the PodEvictor (one that calls deletePods)